tests/unit/querybuf.tcl

```diff
@@ -78,6 +119,11 @@ start_server {tags {"querybuf slow"}} {
         $rd write "*3\r\n\$3\r\nset\r\n\$1\r\na\r\n\$1000000\r\n"
         $rd flush
         
+        after 200
```

Comment:
we need to wait for redis to read that incomplete command? maybe we better use wait_for to avoid timing issues.

same in theory applies for the `after 20` below, but i'm not sure we can detect it.
maybe by looking at the `argv-mem` and `qbuf` fields in CLIENT LIST?

Reply:
> we need to wait for redis to read that incomplete command? maybe we better use wait_for to avoid timing issues.

fixed in [`e5a4a67` (#337)](https://github.com/sundb/redis/pull/337/commits/e5a4a67f1273da817d3029296f35adc79d6369c6).

> same in theory applies for the `after 20` below, but i'm not sure we can detect it. maybe by looking at the `argv-mem` and `qbuf` fields in CLIENT LIST?

IIRC, the `after 20` is used to verify that the next client cron doesn't shrink the client's query buffer.
in [`e5a4a67` (#337)](https://github.com/sundb/redis/pull/337/commits/e5a4a67f1273da817d3029296f35adc79d6369c6), we turn on cron before `after`, so `after 120` should be more reasonable.

Reply:
well, that can still have a race condition, if we want to wait for cron, we can maybe use a wait_for on some new tick metric. but we don't do that elsewhere. i suppose 120 is good.

Reply:
yes, but in theory it's hard to run for more than 2 seconds, I'll check it from daily CI.

Reply:
no need. if we'll ever see it fail we'll adjust. we have similar after 120 in other places.